src/components/CreateTemplate/index.tsx

```diff
@@ -0,0 +1,228 @@
+            <FormControl>
+              <FormLabel>Court Preview:</FormLabel>
+              <Flex
+                fontSize="2rem"
+                fontWeight="500"
+                width="20rem"
+                height="11rem"
+                backgroundColor="orange"
+                justifyContent="center"
+                alignItems="center"
+                margin="1.8rem auto"
+              >
+                Court Image
+              </Flex>
+            </FormControl>
+            <Flex gap="1.5rem" justifyContent="center" flexWrap="wrap">
+              <Badge margin="1rem" colorScheme="green">
+                {courtType}
+              </Badge>
+              <Badge margin="1rem">{selectedCourt.courtName}</Badge>
+            </Flex>
+            <Flex>
+              <FormControl
+                width="50%"
+                marginTop="1rem"
+                isRequired
+                isInvalid={inputError.courtNameFullErr}
+              >
+                <FormLabel marginBottom="1rem">Template Court Name:</FormLabel>
+                <Input
+                  role="courtNameInput"
+                  placeholder="Court name"
+                  width="15rem"
+                  onChange={checkNameLength}
+                  autoFocus
+                  ref={courtNameRef}
+                />
+              </FormControl>
+              <Box width="50%">
+                <Text margin="1rem 0px" fontSize="medium" fontWeight="500">
+                  Publisher:
+                </Text>
+                <Flex alignItems="center">
+                  <Icon as={FaUserCircle} fontSize="2.5rem" marginRight="1.8rem" />
+                  <Text fontSize="large" fontWeight="500">
+                    {userFullName}
+                  </Text>
+                </Flex>
+              </Box>
+            </Flex>
+            <Text
+              color="crimson"
+              visibility={
+                inputError.courtNameFullErr || inputError.courtNameNullErr ? "visible" : "hidden"
+              }
+              fontSize="0.8rem"
+            >
+              {inputError.courtNameFullErr
+                ? inputErrorMsg.nameFullErrMsg
+                : inputErrorMsg.nameNullErrMsg}
+            </Text>
+            <FormControl marginTop="1rem">
+              <FormLabel marginBottom="1rem">Description:</FormLabel>
+              <Textarea
+                height="9.5rem"
+                placeholder={`Description: maximum ${maxDescriptionLen} words`}
+                onChange={handleTextAreaLenChange}
+                resize="none"
+                ref={descriptionRef}
+              />
+            </FormControl>
+            <Text role="wordCount" color={textAreaLen < maxDescriptionLen ? "black" : "crimson"}>
+              {inputError.descriptionOverLimit
+                ? inputErrorMsg.descriptionLenErrMsg
+                : `${textAreaLen}/${maxDescriptionLen} words`}
+            </Text>
```

Comment:
What is the purpose of using FormControl here?
<img width="837" alt="image" src="https://user-images.githubusercontent.com/61776449/196412673-382a1ec9-d334-420f-9a1c-e6d47c5098f0.png">

Reply:
Deleted